tools/gen-intrinsics-specs.py

```diff
@@ -78,6 +89,9 @@ def rst_literal_quote(mapping, source_syntax):
     elif source_syntax == "markdown":
         indented_lines = [f"`{line.strip()}`" for line in lines]
         return '<br>'.join(indented_lines)
+    elif source_syntax == "pdf":
+        indented_lines = [f"`{line.strip()}`" for line in lines]
```

Comment:
We don't need to indent the lines... We just need to split `"a; b"` into `["a", " b"]` (that's already done in line 83) and then use `.join` to flue them together, wrapped in `<three times that symbo>`.